### PR TITLE
Joy

### PR DESCRIPTION
**Pull request by JoyWang0801**
- changed frontend Dockerfile
- modified README.md

### CODE REVIEW

File: frontend/.gitignore
Comment (YuOrii) [inline frontend/.gitignore:1]: why is frontend folder deleted?

File: readme.md [hunk 1]
```diff
@@ -51,19 +51,19 @@ cd frontend
...
 
 ```
 
-3\. Once the dependencies are installed, you can start the React development server:
+3\. After the image has been built, you can run the frontend react app:
 
 ```sh
 
-npm start
+docker run hrease-frontend
```
Comment (YuOrii) [inline readme.md]: maybe we can do a `docker-compose up` for both front and back end?

**Review by YuOrii — CHANGES_REQUESTED** [review]

File: frontend/.gitignore
Comment (JoyWang0801) [reply frontend/.gitignore:1]: wait what? let me double check

Comment (JoyWang0801) [reply frontend/.gitignore:1]: fixed

File: readme.md (hunk 1 above)
Comment (JoyWang0801) [reply readme.md]: Working on this, will let you know once it's done.

Comment (JoyWang0801) [reply readme.md]: done

File: compose.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+services:
+  frontend:
+    build: ./frontend
+    ports:
+      - "3000:3000"
+
+  backend:
+    build: ./backend
+    ports:
+      - "8080:8080"
```
Comment (YuOrii) [inline compose.yaml:10]: Please check that port binding is correct. Default port for Pocketbase is 8090

Comment (YuOrii) [reply compose.yaml:10]: Check Pocketbase Admin UI is accessible via Docker image

**Review by YuOrii — CHANGES_REQUESTED** [review]

File: compose.yaml (hunk 2 above)
Comment (YuOrii) [reply compose.yaml:10]: Looks good

**Pull request merged**